src/charm.py

```diff
@@ -167,6 +168,22 @@ def __init__(self, *args):
 
         self.catalog = CatalogueConsumer(charm=self, item=self._catalogue_item)
 
+        self._mesh = ServiceMeshConsumer(
+            self,
+            policies=[
```

Comment:
afaik the [grafana-source](https://github.com/canonical/alertmanager-k8s-operator/blob/ccc9f5c31d210318de43534a04718928dc7d9951/charmcraft.yaml#L77-L81) relation will allow grafana's workload to GET (and maybe POST?) alertmanager's workload, so we'd need a policy for that